community.main/pom.xml

```diff
@@ -23,24 +23,9 @@
         <developerConnection>${project.scm.connection}</developerConnection>
     </scm>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.kaazing</groupId>
-            <artifactId>community.license</artifactId>
-            <!-- We really want -->
-            <!--     <version>{project.version}</version> -->
-            <!-- but due to inheritance rules, this won't work. -->
-            <version>develop-SNAPSHOT</version>
```

Comment:
Sorry, I didn't understand the more subtle part of your first comment. Thanks for clarifying, I'll respond in the other file instead.